src/rosdiscover/interpreter/interpreter.py

```diff
@@ -68,7 +69,8 @@ def launch(self, filename: str) -> None:
                            namespace=node.namespace,  # FIXME
                            launch_filename=node.filename,
                            remappings=remappings,
-                           args=args)
+                           args=args,
+                           config=configuration)
```

Comment:
We could drop the `config` parameter if `configuration` becomes a property of `Interpreter`.